src/main/java/bio/terra/pdao/bigquery/BigQueryPdao.java

```diff
@@ -1002,6 +1107,36 @@ private SnapshotMapColumn lookupMapColumn(Column toColumn, SnapshotMapTable mapT
         return null;
     }
 
+    public void softDeleteRows(Dataset dataset,
```

Comment:
This is another method that is only used in test. I don't think it should be here.

Reply:
Leaving for now because it will be used for upserts